make/photon/prepare/templates/docker_compose/docker-compose.yml.jinja

```diff
@@ -69,7 +69,7 @@ services:
         tag: "registry"
   registryctl:
     image: goharbor/harbor-registryctl:{{version}}
-    container_name: registryctl
+    container_name: harbor-registryctl
```

Comment:
Harbor has only implemented registryctl, while the rest remain as vanilla distribution, Redis and Nginx.

Reply:
> Harbor has only implemented registryctl, while the rest remain as vanilla distribution, Redis and Nginx.

This is the name of the container, which can be confusing if other projects also have nginx, redis, etc. The purpose is to distinguish which project the container belongs to.